### DISCUSSION TITLE
flashing ESP32CAM, but no wifi network

### DISCUSSION THREAD

**Discussion by defaliz** [arendst/Tasmota#14779] [Q&A] [answered]
hi,
this is the third esp32cam that i flash with esptool by using this page https://tasmota.github.io/docs/ESP32/#flashing
for one esp32 i have a problem..
no problem to flash, but when reseting no wifi tasmota is visible...
i re-began several time flashing with same issue..
with screen i can see this :

ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DOUT, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12352
load:0x40080400,len:2912
entry 0x400805c4

do you know why there is no wifi ?
regards

**Comment by sfromis** [comment]
What's your power source? It may not reliably connect on the USB/serial adapter used for flashing. And if you do not see any Tasmota messages after what you posted, you may not have flashed correctly.

**Comment by defaliz** [comment, marked as answer]
I use power source of my pc via usb..
Do i have to use external power ?

**Reply by barbudor** [reply]
Could be.
Depends on your PC

Alternatively to remove any doubt on your esptool command line, you can try to flash using https://tasmota.github.io/install

![image](https://user-images.githubusercontent.com/2996042/153052913-a9c0fa25-b1bf-433f-96ec-7ddce5a673c5.png)

**Reply by sfromis** [reply]
To be "more clear" about what I posted earlier. You should see Tasmota messages (starting with a time stamp) on serial console, before it is even trying to connect to wifi. You posted no such messages, and if none are there, you likely have done the flashing the wrong way.

**Reply by defaliz** [reply]
many thanks Barbudor ! i use your link (tasmota serial web) and that works !!!
so the cli commands with esptool was not good..
best regards 
Alain
